lib/awful/spawn.lua

```diff
@@ -130,20 +130,19 @@
 --
 -- **Default applications**:
 --
--- If the intent of the spawning is opening a document, then it is recommended
--- to use the standard command. How the default application is selected is
+-- If the intent is to open a document, then it is recommended to use the
+-- following standard command. How the default application is selected is
 -- defined in the [Shared MIME-info Database](https://specifications.freedesktop.org/shared-mime-info-spec/shared-mime-info-spec-latest.html)
 -- specification. The `xdg-utils` package provided by most distributions
--- will provide the `xdg-open` command:
+-- provides the `xdg-open` command:
 --
---    awful.spawn("xdg-open /path/to/file")
+--    awful.spawn({"xdg-open", "/path/to/file"})
```

Comment:
Is this change ok?

I know that everyone thinks of commands as strings. However, the kernel (the `execve` syscall) wants an array of strings instead. Directly giving such an array to awesome avoids all kinds of word-splitting problems.

Sadly, this cannot really be used in the other examples...
(`noisy` could be turned into `noisy = { "bash", "-c", [[lots of text]] }`, but perhaps in this case that is more confusing than helping)